photon-core/src/main/java/org/photonvision/common/util/ShellExec.java

```diff
@@ -72,6 +72,32 @@ public int executeBashCommand(String command, boolean wait) throws IOException {
         return retcode;
     }
 
+    /**
+     * Execute a bash command. We can handle complex bash commands including multiple executions (; |
+     * and ||), quotes, expansions ($), escapes (\), e.g.: "cd /abc/def; mv ghi 'older ghi '$(whoami)"
+     * This runs the commands with the default logging.
+     *
+     * @param command Bash command to execute
+     * @return true if bash got started, but your command may have failed.
+     */
+    public int executeBashCommandNoLog(String command, boolean wait) throws IOException {
```

Comment:
Is this the same as the above, just without the logger print?

Reply:
It is. I did this temporarily to get things working without messing with the other command. Now that it's working I'll refactor to avoid duplicate code.

Reply:
Now it's overloaded.